src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java

```diff
@@ -497,28 +497,24 @@ private AdsDataProvider() {
             "adtest",
             "SyntheticTests/Interstitial",
             new Dimension(1920, 1080),
-            new Dimension(600, 590),
             true
         },
         {
             "adtest",
             "SyntheticTests/Interstitial/NotScalable",
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "SyntheticTests/Interstitial/NotScalable" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)